guide/docs/interactions/context-menus.mdx

```diff
@@ -0,0 +1,72 @@
+---
+description: They are available under Apps in user/message context menus, responding with the resolved user or message on which the action was taken.
+---
+
+# Context Menus
+
+These application commands are situated under the "Apps" section when you right-click a user or a message. They act like interactions, except they take a <DocsLink reference="disnake.User" /> or <DocsLink reference="disnake.Message" /> object along with <DocsLink reference="disnake.ApplicationCommandInteraction" />.
+
+<!-- prettier-ignore -->
+<br />
+<p align = "center">
+	<img src={require("./images/context-menus-intro.png").default} alt="Context Menu Introduction" width="70%"/>
+</p>
+<br />
+
+:::note
+
+For more specific annotations, you can use <DocsLink reference="disnake.UserCommandInteraction" /> or <DocsLink reference="disnake.MessageCommandInteraction" /> instead of <DocsLink reference="disnake.ApplicationCommandInteraction" /> for the respective commands.
+
+:::
+
+## User commands
+
+<DiscordMessages>
+	<DiscordMessage profile="bot">
+		<div slot="interactions">
+			<DiscordInteraction profile="user" command={true}>
+				User Info
+			</DiscordInteraction>
+		</div>
+		Username: AbhigyanTrips#2474
+		<br />
+		Bot: False
+	</DiscordMessage>
+</DiscordMessages>
+<br />
+
+The function below should have only one required argument, which is an instance of <DocsLink reference="disnake.ApplicationCommandInteraction" />.
```

Comment:
might be a bit confusing, since this mentions "only one required argument" while using two in the following example